.env.alfajores

```diff
@@ -35,7 +35,7 @@ CELOSTATS_BANNED_ADDRESSES=""
 CELOSTATS_RESERVED_ADDRESSES=""
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="celoprod.azurecr.io/oracle"
-ORACLE_DOCKER_IMAGE_TAG="3cc6459db821f03ce0df12d2891b3ff4b34e3f05"
+ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```

Comment:
@martinvol Is this update to get an oracle version with the updated Coinbase's liveness check?

Reply:
Yes

Reply:
Got it, thanks. I've updated all the env files.